pkg/client/client.go

```diff
@@ -31,7 +31,7 @@ func (c *Client) Do(req *http.Request) (*http.Response, error) {
 }
 
 // Send an event to the agent daemon server.
-func (c *Client) Send(event eventsapi.EventV2) (*http.Response, error) {
+func (c *Client) Send(event eventsapi.Event, eventVersion eventsapi.EventVersion) (*http.Response, error) {
```

Comment:
Any reason we need to explicitly pass `eventVersion` as a parameter?  It seems like we could add a `Version` method to the `Event` interface like so: 

```go
type Event interface {
	GetRoutingKey() string
	Validate() error
	Version() EventVersion
}
```

Reply:
Nice, this is a lot cleaner, fixed here: 2051614